src/SubscriptionStatus.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+namespace Landingi\Core;
+
+use Exception;
+use function in_array;
+
+final class SubscriptionStatus
```

Comment:
Ja zazwyczaj dodaje jeszcze metody `is` np: `isPastDue`. To dobre podejście?

Reply:
Właśnie myślałem nad czterema checkerami, ale doszedłem do wniosku, ze jak beda potrzebne to sie doda :)